src/lib.rs

```diff
@@ -2803,6 +2821,12 @@ impl Build {
                     format!("arm-kmc-eabi-{}", gnu)
                 } else if target.arch == "aarch64" && target.vendor == "kmc" {
                     format!("aarch64-kmc-elf-{}", gnu)
+                } else if target.os == "nto" {
+                    if self.cpp {
+                        "q++".to_string()
+                    } else {
+                        "qcc".to_string()
+                    }
```

Comment:
Nit: Would be nice with a note here saying that `qcc` is the only supported compiler (and possibly linking to this PR for context).

Reply:
I've added only a short remark here as I feel it helps more in `fn add_default_flags` where we assume that qcc/q++ is used (when adding the special arguments).